src/current/v25.2/architecture/replication-layer.md

```diff
@@ -146,51 +156,42 @@ Because leases do not expire until a node disconnects from a cluster, leaseholde
 
 A table's meta and system ranges (detailed in the [distribution layer]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#meta-ranges)) are treated as normal key-value data, and therefore have leases just like table data.
 
-However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
-
-#### Leader leases
+Unlike table data, system ranges use expiration-based leases; expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+Expiration-based leases are used:
 
+- By some system ranges, as described above.
```

Comment:
Can we cite the exact section this is referring to?

```suggestion
- By some system ranges, as described above.
```

Reply:
I realized we just explained this in the previous paragraph (where it says "Unlike table data, system ranges use expiration-based leases") so referring back to something so recent is probably not necessary

instead i've updated this last bit to read 

> Expiration-based leases are also used temporarily during operations like lease transfers, until the new Raft leader can be fortified based on store liveness as described in [Leader leases](#leader-leases).